src/sqlfluff/core/dialects/dialect_ansi.py

```diff
@@ -1720,7 +1829,7 @@ class UpdateStatementSegment(BaseSegment):
     UPDATE <table name> SET <set clause list> [ WHERE <search condition> ]
     """
 
-    type = "delete_statement"
+    type = "dupdate_statement"
```

Comment:
dupdate? `update_statement`?

Reply:
I tried to correct the false type and failed...haha^^